docs/admin/audit-logs.md

```diff
@@ -0,0 +1,24 @@
+# Audit Logs
+
+Audit logs is an enterprise feature that allows **Admins** and **Auditors** to monitoring what is happening on their deployment.
+
+## Tracked Events
+
+The following events are tracked by this feature:
+
+- Create
+- Update
+- Delete
+
+For the following resources:
+
+- GitSSHKey
+- Template
+- TemplateVersion
+- Workspace
+- APIKey
+- User
```

Comment:
Ooooo I see it now... Hmm, I wonder if we can structure this better. Maybe something like:

This feature tracks create, update and delete events for the following resources:
- list

Reply:
Makes sense.